packages/lesswrong/lib/collections/bookmarks/fragments.ts

```diff
@@ -0,0 +1,22 @@
+import { frag } from "@/lib/fragments/fragmentWrapper";
+
+export const BookmarksDefaultFragment = () => frag`
```

Comment:
pretty sure this one is generated by default in `defaultFragments.ts` and you shouldn't define it again?